src/definitions/elements/label.less

```diff
@@ -899,7 +899,6 @@ a.ui.active.label:hover::before {
     .ui[class*="bottom pointing"].label::before,
     .ui[class*="pointing below"].label::before {
         border-width: 0 @borderWidth @borderWidth 0;
-        top: auto;
```

Comment:
see https://github.com/fomantic/Fomantic-UI/pull/2594/commits/52df34588293a7ceb3d372c9700ac201f0ba979a, maybe there is some typo in the code

Reply:
This fix here is ok, top:100% wins all the time